trigger/payloads.go

```diff
@@ -451,7 +451,20 @@ func getFailedTestsFromTemplate(t task.Task) ([]task.TestResult, error) {
 	for i := range t.LocalTestResults {
 		if t.LocalTestResults[i].Status == evergreen.TestFailedStatus {
 			testResult := t.LocalTestResults[i]
-			testResult.URL = settings.Ui.Url + testResult.URL
+			if testResult.URL != "" {
+				logURL, err := url.Parse(testResult.URL)
```

Comment:
I still don't see, though: why do we need to parse the URL? If it's set (`!= ""`) can't we just provide that? Are there any testresults that don't have a host?

Reply:
I left the parsing in as a sanity check, in case there's a scenario we aren't aware of where the URL is a relative one. Realistically we should either be getting an absolute one or a log ID